pyproject.toml

```
@@ -123,7 +123,7 @@ dataflow = [
 # GPU custom kernel dependency.
 gpu = [
     "triton==2.1.0",
-    "jax[cuda12_pip]==0.4.30",
+    "jax[cuda12_pip]==0.4.33",
```

```suggestion
    "jax[cuda12]==0.4.33",
```

https://jax.readthedocs.io/en/latest/installation.html#pip-installation-nvidia-gpu-cuda-installed-via-pip-easier

---

I think originally the guide used `cuda12_pip` but at some point it was changed. Note that today both work.

---

Let's make that change on the next Jax upgrade